### PR TITLE
Products page

### PR DESCRIPTION
**Pull request by Loges2108**

### CODE REVIEW

File: src/common/component/reusablecard/ReusableCard.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import React, { useState } from "react";
+import { Collection} from "../../../interface/type"; 
+import { Link } from "react-router-dom";
+import { Container, Typography, Grid, Card, CardActionArea, CardContent, CardMedia, Divider } from "@mui/material";
+
+interface Props {
+  collection: Collection;
+}
+
+const CollectionPage: React.FC<Props> = ({ collection }) => {
```
Comment (sathyaprabaS) [inline src/common/component/reusablecard/ReusableCard.tsx]: for what u are using

File: src/common/component/reusablecard/ReusableCard.tsx [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+import MenuItem from "@mui/material/MenuItem";
+
+interface Props {
+  collection: Collection;
+}
+
+const CollectionPage = (props: Props) => {
+  const { collection } = props;
+  const [hoveredProductId, setHoveredProductId] = useState<string | null>(null);
+  const [expanded, setExpanded] = useState(false);
```
Comment (sathyaprabaS) [inline src/common/component/reusablecard/ReusableCard.tsx]: change the name properly

File: src/common/component/reusablecard/ReusableCard.tsx [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+  const [sortOption, setSortOption] = useState<string>('');
```
Comment (sathyaprabaS) [inline src/common/component/reusablecard/ReusableCard.tsx]: sort option means what give name properly

File: src/common/component/reusablecard/ReusableCard.tsx (hunk 1 above)
Comment (Loges2108) [reply src/common/component/reusablecard/ReusableCard.tsx]: resolved

File: src/common/component/reusablecard/ReusableCard.tsx (hunk 2 above)
Comment (Loges2108) [reply src/common/component/reusablecard/ReusableCard.tsx]: resolved

File: src/common/component/reusablecard/ReusableCard.tsx (hunk 3 above)
Comment (Loges2108) [reply src/common/component/reusablecard/ReusableCard.tsx]: i changed name

**Pull request merged**